packages/dataviews/src/types.ts

```diff
@@ -164,3 +164,78 @@ export interface ViewList extends ViewBase {
 }
 
 export type View = ViewList | ViewBase;
+
+interface ActionBase {
+	/**
+	 * The unique identifier of the action.
+	 */
+	id: string;
+
+	/**
+	 * The label of the action.
+	 */
+	label: string;
+
+	/**
+	 * The icon of the action. (Either a string or an SVG element)
+	 * This should be IconType from the components package
+	 * but that import is breaking typescript build for the moment.
+	 */
+	icon?: any;
```

Comment:
So it turns out that it was the import of `IconType` from the components package that was breaking the build in this PR. Using any temporarily should fix it.

Reply:
😆 Just when #61486 is almost ready.